processing/src/test/java/io/druid/segment/IndexMergerTest.java

```diff
@@ -346,65 +406,6 @@ public void testPersistEmptyColumn() throws Exception
   }
 
   @Test
-  public void testMergeRetainsValues() throws Exception
```

Comment:
Its code place had been changed so it looks like this ut been removed, you can check out here:
https://github.com/druid-io/druid/pull/2138/files#diff-84cda5dd68d43d141ccedbbb7b3cbfd0